content/docs/dvclive/dvclive-with-dvc.md

```diff
@@ -0,0 +1,147 @@
+# dvclive with DVC
```

Comment:
```suggestion
# Dvclive with DVC
```

Reply:
Should capitalize everywhere or put it in back quotes if it refers to the py package name.